src/pages/verify.tsx

```diff
@@ -26,6 +53,25 @@ const VerifyPage = (): React.ReactElement => {
 
       {getProvider() ? (
         <Page title="Verify Documents">
+          <span className="text-gray-900 mr-3 mb-3" data-testid="page-subtitle">
+            Verify your document on
+          </span>
+          <NetworkSelect />
+          <span className="mr-3" />
```

Comment:
try not to put a empty span with just margin.
if need be, create a <div> instead of <span>, and style it accordingly.

from what i see u can do something like:
```
<div className="flex">
  <div className="text-gray-900 mr-3 mb-3" data-testid="page-subtitle">Verify your document on</div>
  <div className="mr-3">
    <NetworkSelect />
  </div>
</div>
```
i dont know y u need a margin right in the span, but if u can use flexbox to adjust it then u should do it.

Reply:
updated, thank you